services/app-web/src/components/FileUpload/FileItem/FileItem.tsx

```diff
@@ -1,141 +1,42 @@
-                <span className={styles.fileItemBoldMessage}>
-                    Unexpected error. Please remove.
-                </span>
-            </>
-        )
-    } else {
-        return null
-    }
-}
-
-type FileItemProps = {
+type FileListItemProps = {
+    errorRowClass?: string
+    isLoading: boolean
+    isScanning: boolean
+    statusValue: string
     item: FileItemT
-    deleteItem: (item: FileItemT) => void
-    retryItem: (item: FileItemT) => void
+    imageClasses: string
+    documentError: React.ReactElement | null
+    hasRecoverableError: boolean
+    handleDelete: (_e: React.MouseEvent) => void
+    handleRetry: (_e: React.MouseEvent) => void
 }
-export const FileItem = ({
-    item,
-    deleteItem,
-    retryItem,
-}: FileItemProps): React.ReactElement => {
-    const { name, status, file } = item
-    const hasDuplicateNameError = status === 'DUPLICATE_NAME_ERROR'
-    const hasScanningError = status === 'SCANNING_ERROR'
-    const hasUploadError = status === 'UPLOAD_ERROR'
-    const hasUnexpectedError = status === 'UPLOAD_ERROR' && file === undefined
-    const isLoading = status === 'PENDING'
-    const isScanning = status === 'SCANNING'
-
-    const isPDF = name.indexOf('.pdf') > 0
-    const isWord = name.indexOf('.doc') > 0 || name.indexOf('.pages') > 0
-    const isVideo = name.indexOf('.mov') > 0 || name.indexOf('.mp4') > 0
-    const isExcel = name.indexOf('.xls') > 0 || name.indexOf('.numbers') > 0
-    const isGeneric = !isPDF && !isWord && !isVideo && !isExcel
-
-    const imageClasses = classnames('usa-file-input__preview-image', {
-        'is-loading': isLoading || isScanning,
-        'usa-file-input__preview-image--pdf': isPDF,
-        'usa-file-input__preview-image--word': isWord,
-        'usa-file-input__preview-image--video': isVideo,
-        'usa-file-input__preview-image--excel': isExcel,
-        'usa-file-input__preview-image--generic': isGeneric,
-    })
-
-    const handleDelete = (_e: React.MouseEvent) => {
-        deleteItem(item)
-    }
-
-    const handleRetry = (_e: React.MouseEvent) => {
-        retryItem(item)
-    }
-
-    let statusValue = "";
-    if (isLoading) {
-        statusValue = "uploading";
-    } else if (isScanning) {
-        statusValue = "scanning for viruses"
-    } else if (hasDuplicateNameError || hasScanningError || hasUploadError || hasUnexpectedError) {
-        statusValue = "error"
-    }
 
+export const FileListItem = ({
+    isLoading,
+    isScanning,
+    statusValue,
+    item,
+    imageClasses,
+    documentError,
+    hasRecoverableError,
+    handleDelete,
+    handleRetry,
+}: FileListItemProps): React.ReactElement => {
+    const { name } = item
     return (
         <>
             <div className={styles.fileItemText}>
                 <div
                     role="progressbar"
                     aria-valuetext={statusValue}
-                    aria-label={`File status`}
+                    aria-label={`Status of file ${name}`}
```

Comment:
I don't think this change is needed. JAWS will already announce the the file name immediately after the file status: loading because its written on the screen in `<span>{name}</span>`. We are trying not to over use `aria-label` - in this case it will re-announce something  (the file names, and file names are often very lengthy in our use case) that is already being announced.

The only reason we had to change things for the Remove and Retry buttons was because in "button" mode in JAWS it skips over anything that is not a button and it wasn't clear what button went with what file in button mode

Reply:
I made these changes after I merged in main and had failing tests; so I think they came from one of the accessibility tickets?  We can revert in a future PR if it's a problem.